grails-app/migrations/2018-10-01-v500-job-label-optional.groovy

```diff
@@ -0,0 +1,5 @@
+databaseChangeLog = {
+    changeSet(author: "fho", id: "20181001-label-1") {
+        dropNotNullConstraint(columnDataType: "varchar(255)", tableName: "job", columnName: "label")
```

Comment:
I think there is also a unique index on the column. Can you check if and how it can be removed as well?